packages/mesos-modules/build

```diff
@@ -1,15 +1,29 @@
 #!/bin/bash
+set -o nounset -o pipefail
+
 pushd /pkg/src/mesos-modules/
 ./bootstrap
 popd
 
-source /opt/mesosphere/environment.export
+export LD_LIBRARY_PATH=/opt/mesosphere/lib
```

Comment:
it's better to source environment.export here to get other important things dependencies might set. Sourcing environment.export will also always set LD_LIBRARY_PATH